crates/frontend/src/components/context_form/utils.rs

```diff
@@ -1,73 +1,192 @@
+                    .into_iter()
+                    .map(ConfigType::Dimension)
+                    .collect::<Vec<_>>(),
+            )?;
 
             Ok(json!({
-                op: [
-                    first_operand,
+                "<=": [
+                    first_operand_value,
                     { "var": var },
-                    dimension_val.expect("can't parse dimension value")
+                    third_operand_value
                 ]
             }))
         }
-        _ => {
-            let dimension_val = get_config_value(
+        ConditionOperator::Is => {
+            // Expecting two elements for "Is" condition: one "var" object and one value
+            if filtered_values.len() != 1 {
+                return Err("Invalid number of operands for 'is' condition.".to_string());
+            }
+
+            let value = &filtered_values[0]; // The value after "var"
+            let first_operand_value = get_config_value(
                 var,
-                val,
+                value,
                 &dimensions
                     .into_iter()
                     .map(ConfigType::Dimension)
                     .collect::<Vec<_>>(),
-            );
+            )?;
+
             Ok(json!({
-                op: [
-                    {"var": var},
-                    dimension_val.expect("can't parse dimension value")
+                "==": [
+                    { "var": var },
+                    first_operand_value
                 ]
             }))
         }
+        ConditionOperator::In => {
+            if filtered_values.len() != 1 {
+                return Err("Invalid number of operands for 'in' condition.".to_string());
+            }
+            let value = &filtered_values[0]; // The value after "var"
+            let first_operand_value = get_config_value(
+                var,
+                value,
+                &dimensions
+                    .into_iter()
+                    .map(ConfigType::Dimension)
+                    .collect::<Vec<_>>(),
+            )?;
+
+            Ok(json!({
+                "in": [
+                    { "var": var },
+                    first_operand_value
+                ]
+            }))
+        }
+        ConditionOperator::Has => {
+            if filtered_values.len() != 1 {
+                return Err("Invalid number of operands for 'has' condition.".to_string());
+            }
+            let value = &filtered_values[0]; // The value after "var"
+            let first_operand_value = get_config_value(
+                var,
+                value,
+                &dimensions
+                    .into_iter()
+                    .map(ConfigType::Dimension)
+                    .collect::<Vec<_>>(),
+            )?;
+
+            Ok(json!({
+                "in": [
+                    first_operand_value,
+                    { "var": var }
+                ]
+            }))
+        }
+        ConditionOperator::Other(op) => {
```

Comment:
I guess, since we are clear on not supporting any unknown operator through forms, this particular variant can be dropped in future.